src/CrowdDesign/Content/css/EditProject.css

```diff
@@ -7,29 +15,71 @@
 }
 
 #tblMorphChart {
-    width: 100%;
+    border: 0; 
+    border-collapse:collapse;
 }
 
-#tblMorphChart td {
-    word-wrap: break-word;
+#tblMorphChart th {
+    border-bottom: 7px double black;
+    height: 50px;
 }
 
-#imgSketchThumbnail {
-    width: 50%;
-    height: 50%;
+#tblMorphChart th:first-child {
+    border-right: 7px double black;
 }
 
-.tdAddNewSketch {
-    text-align: left;
-    border: none;
+#tblMorphChart > tbody > tr > td:first-child {
+    border-right: 7px double;
 }
 
-.tdAddNewDimension {
-    border: none;
+#tblMorphChart td:first-child{
+    border-left: 1px solid black;
+}
+
+#tblMorphChart td:last-child{
+    border-right: 1px solid black;
 }
 
 .tdDimension {
-    border-right: double;
+    min-width: 300px;
+    /*min-height: 150px;*/
+    text-align: center;
+    font-size: 0.9em; 
+
+}
+
+.pDimensionDescription {
+    font-size: 0.8em; 
+
+}
+
+#tblMorphChart td{
+    /*can't use margin because it's a table cell*/
+    border-right: none;
+    border-left: none;
+    word-wrap: break-word; 
+    height: 100px;
```

Comment:
Reminder: we should check how smaller devices behave with a fixed height / width